demo/webpack/config.js

```diff
@@ -26,20 +41,16 @@ module.exports = {
       loader: 'raw-loader',
       include: demoSources,
       enforce: 'post'
-    }, {
-      test: /\.js$/,
-      loader: 'transform-loader?brfs-babel'
     }]
   },
 
   resolve: {
     modules: [resolve(rootDir, 'node_modules'), resolve(demoDir, 'node_modules')],
     alias: {
       'deck.gl': libSources,
-      webworkify: 'webworkify-webpack-dropin'
-      // react: resolve('./node_modules/react'),
-      // 'react-dom': resolve('./node_modules/react-dom'),
-      // 'gl-matrix': resolve('./node_modules/gl-matrix/dist/gl-matrix.js')
+      webworkify: 'webworkify-webpack-dropin',
```

Comment:
I recommend adding a comment above every line in a webpack config. They are really hard to read otherwise. E.g. explain why we need webworkify, what does it do...